deploy/install.sh

```diff
@@ -312,11 +315,19 @@ apiRequest() {
   url=$2
   json=$3
 
-  response="$(curl -sS -X "${method}" "${API_URL}${url}" \
-    -H "accept: application/json; charset=utf-8" \
-    -H "Authorization: Api-Token ${API_TOKEN}" \
-    -H "Content-Type: application/json; charset=utf-8" \
-    -d "${json}")"
+  if "$SKIP_CERT_CHECK" = "true"; then
+    response="$(curl -k -sS -X ${method} "${API_URL}${url}" \
+      -H "accept: application/json; charset=utf-8" \
+      -H "Authorization: Api-Token ${API_TOKEN}" \
+      -H "Content-Type: application/json; charset=utf-8" \
+      -d "${json}")"
+  else
+    response="$(curl -sS -X ${method} "${API_URL}${url}" \
+      -H "accept: application/json; charset=utf-8" \
+      -H "Authorization: Api-Token ${API_TOKEN}" \
+      -H "Content-Type: application/json; charset=utf-8" \
+      -d "${json}")"
+  fi
 
   echo "$response"
```

Comment:
Is the reponse you get here speaking for itself or would it make sense to prepend this with some information?

Reply:
This is not really possible since we are using this request for every different api call we are doing within the script. So having proper handling would mean identifying the responses upfront, parsing the response as json and then getting the correct (and probably interesting part) out of it